#! /bin/sh /usr/share/dpatch/dpatch-run
## udev-rules-use-attrs-not-sysfs-and-hp-mkuri-call-fix.dpatch by Daniel T 
## Chen <crimsun@ubuntu.com> and Till Kamppeter <till.kamppeter@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix use of deprecated udev syntax (LP: #491346)

@DPATCH@
diff -urNad hplip-3.9.12~/data/rules/56-hpmud_support.rules hplip-3.9.12/data/rules/56-hpmud_support.rules
--- hplip-3.9.12~/data/rules/56-hpmud_support.rules	2009-12-15 22:20:58.000000000 +0100
+++ hplip-3.9.12/data/rules/56-hpmud_support.rules	2010-02-10 23:38:46.586522868 +0100
@@ -7,8 +7,6 @@
 LABEL="pid_test"
 
 # Check for LaserJet products (0x03f0xx17).
-SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="??17", ENV{hp_model}="$sysfs{product}", ENV{hp_test}="yes"
-
-ENV{hp_test}=="yes", RUN+="bin/sh -c '/usr/bin/hp-mkuri -c &'"
+ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="??17", RUN+="/bin/sh -c 'hp_model=%E{ID_MODEL} /usr/bin/hp-mkuri -c &'"
 
-LABEL="hpmud_rules_end"
\ No newline at end of file
+LABEL="hpmud_rules_end"
